src/core/simple_lru.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+#pragma once
+
+#include <absl/container/flat_hash_map.h>
+
+#include "base/string_view_sso.h"
+
+namespace dfly {
+
+class SimpleLRU {
+  struct Node {
+    base::string_view_sso key;  // key to the table.
+
+    uint32_t prev;
+    uint32_t next;
+
+    uint64_t value;
```

Comment:
Do you intend to use it only for counting?
If so maybe rename to SimpleLruCounter and uint64_t count, it will also make it clearer why the value is not templated

Reply:
done